stories/useSafeState.story.mdx

```diff
@@ -0,0 +1,22 @@
+import { Meta } from '@storybook/addon-docs/blocks';
+
+<Meta title="State/useSafeState" />
+
+# useSafeState
+
+Have you ever been caught on `Can't perform a React state update on an unmounted component.`? Of
+course you do, we're all been there😅  
+Async callback invoked, it tries to set state, but component is already unmounted, that annoying
+warning happens, and you have to track component mount state manually.
+
+`useSafeState` covers your back - it tracks component mount state and not performs `setState` action
+if component is unmounted, otherwise it is the same hook as common `useState`.
```

Comment:
Maybe:

```
`useSafeState` covers your back - it tracks component mount state and does not perform `setState`
if the component is unmounted, otherwise it is the same hook as common `useState`.
```

Reply:
agree